Automattic/jetpack
# Automatically enable Page Cache when migrating from WP Super Cache.

## Proposed changes:
- Automatically enable Page Cache when migrating from WP Super Cache.
- Add tracking for migrations from WP Super Cache to Jetpack Boost.
- Update the banner design for the migration prompt.
- Update the upgrade to boost banner image.
- Implement checks to prevent duplicate migrations or activations.
- Modify the WP Super Cache admin notice to include a 'Try Jetpack Boost' button.
- Fix the 'Try Jetpack' button issue referenced in issue #37747.

### Other information:



## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Install WP Super Cache and set up caching.
- Migrate from WP Super Cache via the UI.
- Verify that Page Cache is automatically enabled in Boost upon migration.
- Check that the website loads correctly and the cache is functioning as expected.
